app/utils/uploadFile.js

```diff
@@ -6,15 +6,22 @@ type File = {
   blob: boolean,
   type: string,
   size: number,
-  name: string,
+  name?: string,
   file: string,
 };
 
-export default async function uploadFile(file: File) {
+type Options = {
+  name?: string,
+};
+
+export const uploadFile = async (file: File | Blob, option?: Options) => {
+  // $FlowFixMe Blob makes life hard
```

Comment:
Flow really started to piss me off in this file. Didn't really feel adding any value so just ignored